generators/generators-commons/src/main/java/uk/gov/justice/services/generators/commons/helper/Actions.java

```diff
@@ -36,7 +39,8 @@ private Actions() {
     }
 
     public static boolean hasResponseMimeTypes(final Action action) {
```

Comment:
It's not clear to me what this method actually does.
Maybe we should choose a better name for it and/or add a javadoc